definitions/json/ComplexTypes.json

```diff
@@ -651,6 +651,48 @@
     "ElementLabel": "Email address",
     "SecurityClassification": "Public"
   },
+  {
+    "ID": "ClaimantRequest",
+    "ListElementCode": "claimant_compensation",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Claimant is reqesting compensation",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "ClaimantRequest",
+    "ListElementCode": "claimant_tribunal",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Claimant is reqesting a tribunal",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "ClaimantRequest",
+    "ListElementCode": "claimant_old_job",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Claimant is reqesting their old job back",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "ClaimantRequest",
+    "ListElementCode": "claimant_another_job",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Claimant wants to get another job with the employer or an associated employer",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "ClaimantRequest",
+    "ListElementCode": "claimant_compensation_amount",
+    "FieldType": "Number",
+    "ElementLabel": "Compensation Amount Requested",
```

Comment:
is this meant to be capitalised?

Reply:
Not sure if this comment is in the right place but as per the scotland definition we will use the Upper case event ids